components/datetime/src/format/neo.rs

```diff
@@ -952,7 +994,7 @@ impl RawDateTimeNames {
                 ///// Textual symbols /////
                 FieldSymbol::Era => {
                     self.load_year_names(
-                        year_provider.ok_or(Error::MissingNames(field))?,
+                        year_provider.ok_or(LoadError::MissingNames(field))?,
```

Comment:
Thought: With the work I'm doing to consolidate the neo formatter types, this error case _might_ go away which would be nice.